manticore/ethereum/manticore.py

```diff
@@ -384,6 +384,8 @@ def __init__(self, procs=10, workspace_url: str=None, policy: str='random'):
         self._executor.subscribe('on_symbolic_sha3', self._on_symbolic_sha3_callback)
         self._executor.subscribe('on_concrete_sha3', self._on_concrete_sha3_callback)
 
+        self._initargs_id = -1
```

Comment:
the better alternative to this is using `avoid_collisions` when calling `new_array` (in `make_symbolic_buffer` ). that'll take care of automatically appending a number to the symbol name. we might need to add an optional argument to `make_symbolic_buffer` to allow the INITARGS caller to specify that it wants to avoid collisions. otherwise it should default to being off (default to not avoiding collisions, and raising an error)

Reply:
Great idea, added!